test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -186,26 +196,32 @@ const userVerificationAppLinkBiometricsError = RequestMock()
     res.setBody(identifyWithUserVerificationBiometricsErrorMobile);
   });
 
-fixture('Device Challenge Polling View for user verification and MFA with the Loopback Server, Custom URI and Universal Link approaches');
+fixture('Device Challenge Polling View for user verification and MFA with the Loopback Server, Custom URI and Universal Link approaches')
+  .meta('v3', true);
 
 async function setup(t) {
   const deviceChallengePollPage = new DeviceChallengePollPageObject(t);
   await deviceChallengePollPage.navigateToPage();
+  await t.expect(deviceChallengePollPage.formExists()).eql(true);
   return deviceChallengePollPage;
 }
 
 async function setupLoopbackFallback(t) {
   const deviceChallengeFalllbackPage = new IdentityPageObject(t);
   await deviceChallengeFalllbackPage.navigateToPage();
+  await t.expect(deviceChallengeFalllbackPage.formExists()).eql(true);
   return deviceChallengeFalllbackPage;
 }
 
 test
   .requestHooks(loopbackSuccessLogger, loopbackSuccesskMock)('in loopback server approach, probing and polling requests are sent and responded', async t => {
     const deviceChallengePollPageObject = await setup(t);
     await t.expect(deviceChallengePollPageObject.getBeaconClass()).contains(BEACON_CLASS);
-    await t.expect(deviceChallengePollPageObject.getHeader()).eql('Verifying your identity');
-    await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().exists).eql(false);
+    await t.expect(deviceChallengePollPageObject.getFormTitle()).eql('Verifying your identity');
+    // in v3 all cancel buttons are the same so skip this assertion
+    if (!userVariables.v3) {
+      await t.expect(deviceChallengePollPageObject.getFooterCancelPollingLink().exists).eql(false);
+    }
```

Comment:
I remember Lee was working on another spec like this, and I thought it was resolved by [this](https://github.com/okta/okta-signin-widget/pull/3019/files#diff-81d187c79cdc010d9b0a45dc6ef0d3602be29d138652745a88b8fd24f996a628R128-R140) change, but looking at it, I think it is just missing using the [expected translation key](https://github.com/okta/okta-signin-widget/blob/0.1/src/v2/view-builder/views/device/DeviceChallengePollView.js#L61). Then, I think [this](https://github.com/okta/okta-signin-widget/blob/0.1/test/testcafe/framework/page-objects/DeviceChallengePollPageObject.js#L37-L42) page object function can be reverted back to just looking for `Cancel and take me to sign in` (without using the v3 conditional statement): 
```
getFooterCancelPollingLink() {
    return this.form.getLink('Cancel and take me to sign in');
  }
```

Then I think the tests that use the v3 boolean can continue to check for the existence of the link (or non existence depending on the test). 

I hope this makes sense 😭

Reply:
After discussion, we are actually trying to standardize the cancel link text so this change was intentional for v3.